DFC.ServiceTaxonomy.CustomFields/DFC.ServiceTaxonomy.CustomFields.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OrchardCore.Module.Targets" Version="1.0.0-rc1-12607" />
+    <PackageReference Include="OrchardCore.ContentManagement" Version="1.0.0-rc1-12607" />
+    <PackageReference Include="OrchardCore.ContentTypes.Abstractions" Version="1.0.0-rc1-12607" />
+    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.0.0-rc1-12607" />
```

Comment:
Can you change the version to latest, i.e. 1.0.0-rc1-*.
That will probably fix the build as a bonus 😄.